Makefile

```
@@ -891,6 +891,7 @@ $(DEST_DIR)/README: $(SRC_DIR)/README
 # Note: pel--options.el lazily requires some other pel files but they are NOT
 #       included in the dependency as it would create a circular make dependency.
 bin/pel-lint.elc:         pel--base.elc
+bin/pel-lint-delclarations.elc:  pel-elcode.elc
```

checkmake (0.3.2)

[warning] Target "bin/pel-lint-delclarations.elc" should be declared PHONY.

(phonydeclared)